controls/mousestates/cMouseNormalState.cpp

```diff
@@ -1,8 +1,25 @@
 #include <algorithm>
-#include "d2tmh.h"
+
+#include "controls/cGameControlsContext.h"
+#include "controls/cMouse.h"
+
+#include "utils/cRectangle.h"
+#include "gameobjects/units/cUnit.h"
+#include "gameobjects/structures/cAbstractStructure.h"
+#include "player/cPlayer.h"
+#include "sidebar/cSideBar.h"
+
+#include "cGame.h"
+
+// instead of using d2tmc.h
+extern cGame		  game;
+extern cUnit          unit[MAX_UNITS];
 
 #include "cMouseNormalState.h"
```

Comment:
A tip when working with includes: include the header that belongs to the cpp file (so this one) first. Then when compiling cMouseNormalState.cpp, it's checked if the cMouseNormalState.h header can be included separately, so without including other stuff first. If we do this well, then inclusion order no longer matters.